### ISSUE TITLE
Issue with web worker code in minified code

### ISSUE THREAD

**Issue by hckr** [juanelas/bigint-crypto-utils#3]
In your code you use a check for existence of `process.browser` to detect, if the code is run in browser vs. Node (e.g. [here](https://github.com/juanelas/bigint-crypto-utils/blob/ace7d479eef105225d9a9b54a65a53dd9e7c5325/src/main.js#L325). It doesn't work in Firefox which doesn't have such an object defined.

I've changed all occurences of `process.browser` with `navigator` (which should be defined in browsers, but not in Node) and async generation of primes started working correctly.

I don't open a pull request as there might be better ways to detect Node/browser environment.

**Comment by juanelas** [comment]
Hello,
process.browser is a build-time variable in the src/main.js file. In a browser you shouldn't be loading that file but the appropriate one in the dist directory. Please check that you are using the right file. You can also build the js file your self by running npm build in the project base directory.

**Comment by hckr** [comment]
Hi,

Right, I was in a hurry and chose the wrong file.

There is an error, however, and I noticed it only today when I checked in Chrome. Firefox didn't show an error in the worker code, which exists in the minified version, [see this line](https://github.com/juanelas/bigint-crypto-utils/blob/743b48307233b0bb84bb4f2844fa02994e6598ae/dist/bigint-crypto-utils-latest.browser.js#L457). Names of functions are minified, but the code in `workerCode` string stays the same.

**Comment by juanelas** [comment]
I'm sorry hckr but I don't know how to reproduce that issue. I'm testing the minified version in chrome and firefox and, in my case, it just works.  Could you please send me more details? Thanks in advance!

**Comment by hckr** [comment]
Here's a minimal example, doesn't work both in Firefox and Chromium:

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Primes</title>
    <meta charset="UTF-8" />
  </head>
  <body>
    <div id="syncPrime">result of primeSync() will show up here</div>
    <div id="asyncPrime">result of prime() will show up here</div>

    <script src="src/bigint-crypto-utils-latest.browser.min.js"></script>
    <script>
    (async () => {
      document.getElementById(
        "syncPrime"
      ).innerHTML = await bigintCryptoUtils.primeSync(123);
      document.getElementById(
        "asyncPrime"
      ).innerHTML = await bigintCryptoUtils.prime(123);
    })();
    </script>
  </body>
</html>
```

Run in Chromium to see the errors:
![errors in chromium console](https://user-images.githubusercontent.com/1078650/70611113-f8552d00-1c04-11ea-9131-9fd64590aa84.png)

**Comment by juanelas** [comment]
OK. My browser tests were with the mod.min version. This seems definitely a bug. I reopen the issue in order to fix it when I find the time. Thank you very much!

**Comment by juanelas** [comment]
In the meantime (don't know when I should be able to fix it), you could use the mod minified version as (I've tested it and it works):
```html
<!DOCTYPE html>
<html>
  <head>
    <title>Primes</title>
    <meta charset="UTF-8" />
  </head>
  <body>
    <div id="syncPrime">result of primeSync() will show up here</div>
    <div id="asyncPrime">result of prime() will show up here</div>
    <script type="module">
      import * as bigintCryptoUtils from 'src/bigint-crypto-utils-latest.browser.mod.min.js'
      (async () => {
        document.getElementById(
          "syncPrime"
        ).innerHTML = await bigintCryptoUtils.primeSync(123);
        document.getElementById(
          "asyncPrime"
        ).innerHTML = await bigintCryptoUtils.prime(123);
      })();
    </script>
  </body>
</html>
```

**Comment by juanelas** [comment]
Finally fixed in version 2.5.0.
